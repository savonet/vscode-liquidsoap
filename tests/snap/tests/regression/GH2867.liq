def f() =
  output.file(
    %mp3(stereo=false),
    "/tmp/bla",
    mean(blank()))

  test.pass()
end

test.check(f)
