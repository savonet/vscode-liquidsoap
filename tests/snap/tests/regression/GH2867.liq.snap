>def f() =
#^^^ source.liquidsoap keyword.control.liquidsoap
#   ^^^^^^^ source.liquidsoap
>  output.file(
#^^^^^^^^^^^^^^^ source.liquidsoap
>    %mp3(stereo=false),
#^^^^^^^ source.liquidsoap
#       ^ source.liquidsoap constant.numeric.integer
#        ^^^^^^^^^^^^^^^^ source.liquidsoap
>    "/tmp/bla",
#^^^^ source.liquidsoap
#    ^ source.liquidsoap string.quoted.double.liquidsoap
#     ^^^^^^^^ source.liquidsoap string.quoted.double.liquidsoap
#             ^ source.liquidsoap string.quoted.double.liquidsoap
#              ^^ source.liquidsoap
>    mean(blank()))
#^^^^^^^^^^^^^^^^^^^ source.liquidsoap
>
>  test.pass()
#^^^^^^^^^^^^^^ source.liquidsoap
>end
#^^^ source.liquidsoap keyword.control.liquidsoap
>
>test.check(f)
#^^^^^^^^^^^^^^ source.liquidsoap
>